src/sentry/api/endpoints/organization_pinned_searches.py

```diff
@@ -52,6 +53,17 @@ def put(self, request: Request, organization) -> Response:
             visibility=Visibility.OWNER_PINNED,
             values={"query": result["query"], "sort": result["sort"]},
         )
+
+        GroupSearchView.objects.create_or_update(
+            organizaiton=organization,
```

Comment:
`organization`

Reply:
Let's make sure there is a test in place which would catch this, if there isn't already

Reply:
🤦 
Thanks for catching this! Fixing now